classifier/rbdigital.py

```diff
@@ -0,0 +1,97 @@
+from . import *
+
+class RBDigitalAudienceClassifier(FreeformAudienceClassifier):
+
+    @classmethod
+    def target_age(cls, identifier, name):
+        # RBdigital uses 'beginning reader' to refer to books
+        # suitable from birth to age 8, rather than (as is more common)
+        # ages 5 to 8.
+        #
+        # Rather than covering the entire early lifespan of a child,
+        # the normally vague 'childrens' is used here to cover the
+        # time in between 'beginning reader' and 'young adult'
+        if identifier == 'beginning reader':
+            return cls.range_tuple(0,8)
+        elif identifier == 'childrens':
+            return cls.range_tuple(9, 13)
+        return FreeformAudienceClassifier.target_age(identifier, name)
+
+class RBDigitalSubjectClassifier(KeywordBasedClassifier):
+
+    # We have subgenres for fiction in these categories but not for
+    # nonfiction in these categories (thus the None mapping in
+    # 'genres' below).
+    fiction_genres = {
+        'lgbt interest' : LGBTQ_Fiction,
+    }
+
+    genres = {
+        # This isn't true in general but because RBdigital is heavy on
+        # audio content, 'arts and entertainment' is more likely to be
+        # entertainment (e.g. music) than arts.
+        'arts entertainment' : Entertainment,
+        'business economics' : Business,
+        'comics graphic novels' : Comics_Graphic_Novels,
+        'home garden' : Hobbies_Home,
```

Comment:
These are RBdigital subjects where I didn't feel comfortable adding the appropriate tokens to the KeywordBasedClassifier (as I did with e.g. "inspirational nonfiction").